website/docs/r/database_secret_backend_connection.md

```diff
@@ -188,6 +188,10 @@ Exactly one of the nested blocks of configuration options must be supplied.
 * `max_connection_lifetime` - (Optional) The maximum number of seconds to keep
   a connection alive for.
 
+* `username_template` - (Optional) For Vault v1.7+. The username template for vault generate username.
```

Comment:
```suggestion
* `username_template` - (Optional) For Vault v1.7+. The template to use for username generation.
```

Reply:
updated, also using this description to other database, e.g. mysql